### PR TITLE
migrate OPM reports to bootstrap3

### PR DESCRIPTION
**Pull request by biyeun**
@esoergel cc @benrudolph

### CODE REVIEW

**Comment by esoergel** [comment]
@sravfeyn do you still have OPM report data locally?  I lost mine when I switched computers.  This PR looks good, but I'd like to verify that the reports still load properly, particularly the print versions.

**Comment by czue** [comment]
alternatively @lwyszomi or @kkrampa if you have any opm data locally possible to test these?

**Comment by sravfeyn** [comment]
I don't have any data as well. Will try to download some if soldevelo doesn't have it

**Comment by lwyszomi** [comment]
@czue @sravfeyn @esoergel 
We have data localy for these reports, so we can test this.

**Comment by kkrampa** [comment]
Overall, looks good but there are the same problems as in IntraHealth. Look at my comment here: https://github.com/dimagi/commcare-hq/pull/11131#issue-146311982

opm6
opm5
opm4
opm3
opm2
opm1

**Comment by esoergel** [comment]
@kkrampa Thanks for checking that out, did you also look at the printed versions of these reports?

**Comment by esoergel** [comment]
@biyeun do the issues Kamil [mentioned](https://github.com/dimagi/commcare-hq/pull/11110#issuecomment-206433029) from https://github.com/dimagi/commcare-hq/pull/11131#issue-146311982 ring any bells?

**Comment by biyeun** [comment]
@esoergel yep they ring some bells. having a look after wrapping up an HQ notifications PR. Will update shortly this afternoon / evening

**Comment by kkrampa** [comment]
@esoergel @biyeun 
I have just checked print version and there is a problem. Please look at the screen:
printopm

**Comment by czue** [comment]
(removed screenshots)

**Comment by biyeun** [comment]
@kkrampa would you be able to make a PR to this branch since you have data to work with locally? Thank you!

**Comment by biyeun** [comment]
perf, looks like this is g2g @esoergel cc @gcapalbo

File: custom/opm/templates/opm/print_report.html [hunk 1]
```diff
@@ -40,11 +36,11 @@
         tr {
             line-height: 1.3;
         }
+        #loadingReportIssueModal {
+            display: none !important;;
```
Comment (gcapalbo) [inline custom/opm/templates/opm/print_report.html]: (not blocking) does the double semicolon mean something or just a typo?

**Comment by gcapalbo** [comment]
looks good to me, will let @esoergel confirm

**Comment by esoergel** [comment]
Looks good to me, thanks @biyeun and @kkrampa !

**Pull request merged**